### ISSUE TITLE
Update Navigation

### ISSUE THREAD

**Issue by closebracket** [CodheadClub/AwesomeResources#91]
The navigation for the awesome resources list could be much clearer and perhaps generated in a single file so that it can be easily updated. This is an issue which could be resolved by Hacktoberfest participants.

**Comment by superiorsd10** [comment]
@closebracket Can you please describe this issue?

**Comment by closebracket** [comment]
> @closebracket Can you please describe this issue?

Certainly, at the moment the repository contains multiple markdown files each containing resources. Within each of the files is a contents list which is replicated so that a user can traverse the Awesome Resources list easily. 

The first issue is in making that contents list easier to update. One of the suggested ideas to resolve this is to create a template (E.g. "Contents.md") which can be injected into each file via a github action.

The second issue is making the contents list more compact and easier to understand by renaming vague categories after considering the resources contained within and adding sub categories if needed.

If you need any further help, please feel free to ask or join our discord. :)
